napari/layers/points/points.py

```diff
@@ -335,11 +340,30 @@ def size(self, size: Union[None, float]) -> None:
         self._size = size
         if self._update_properties and len(self.selected_data) > 0:
             for i in self.selected_data:
-                self.sizes[i, :] = (self.sizes[i, :] > 0) * size
+                self.sizes[i] = size
+                self._actual_sizes[i] = np.multiply(size, self.anisotropy)
             self._set_view_slice()
         self.status = format_float(self.size)
         self.events.size()
 
+    @property
+    def anisotropy(self) -> Union[tuple]:
+        """tuple: anisotropy factor for each dimension."""
+
+        return self._anisotropy
+
+    @anisotropy.setter
+    def anisotropy(self, anisotropy: Union[None, tuple]) -> None:
+        if anisotropy is None:
+            anisotropy = np.ones(self.dims.ndim)
```

Comment:
The type hint is for `None` or `tuple`, but `anisotropy` is an `ndarray` here.

Reply:
added `np.ndarray`